dpx/src/dpx_pdfnames.rs

```diff
@@ -447,15 +436,10 @@ pub unsafe extern "C" fn pdf_names_create_tree(
     if flat.is_null() {
         name_tree = 0 as *mut pdf_obj
     } else {
-        qsort(
-            flat as *mut libc::c_void,
-            *count as size_t,
-            ::std::mem::size_of::<named_object>() as u64,
-            Some(
-                cmp_key
-                    as unsafe extern "C" fn(_: *const libc::c_void, _: *const libc::c_void) -> i32,
-            ),
-        );
+        ::core::slice::from_raw_parts_mut(
```

Comment:
Same style nitpick here, but also in general for slice, mem, and other function-y modules like those, `use std::slice;` once at the top.